smarkets/streaming_api/framing.py

```diff
@@ -84,9 +87,9 @@ def uleb128_decode(string):
 
     while True:
         try:
-            current_byte = indexbytes(string, position)
+            current_byte = to_decode[position]
```

Comment:
indexbytes from six in Python 3 is just `operator.getitem` (which is assume is just `[]`), in Python 2 it is:
```
def indexbytes(buf, i):
        return ord(buf[i])
```
Ref: https://github.com/django/django/blob/master/django/utils/six.py#L629

So before when it was a string it made sense to do `ord(buf[i])`. But now, it is `bytes`, so just using `[]` is enough, using indexbytes on Python 2 will break with:

```
ERROR: smarkets.tests.streaming_api.framing.test_frame_decode_all(bytearray(b'\x05abcde\x03abc'), (['abcde', 'abc'], ''))
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/adam/workspace/smk_python_sdk/smarkets/tests/streaming_api/framing.py", line 111, in check_frame_decode_all
    eq_(frame_decode_all(byte_array), output)
  File "/home/adam/workspace/smk_python_sdk/smarkets/streaming_api/framing.py", line 57, in frame_decode_all
    decoded = uleb128_decode(to_decode)
  File "/home/adam/workspace/smk_python_sdk/smarkets/streaming_api/framing.py", line 90, in uleb128_decode
    current_byte = indexbytes(to_decode, position)
  File "/usr/local/lib/python2.7/dist-packages/six.py", line 655, in indexbytes
    return ord(buf[i])
TypeError: ord() expected string of length 1, but int found
```

Because it is doing ord() on integer instead of string of length 1.